client/model_create_eks_properties_eks.go

```diff
@@ -14,4 +14,8 @@ package client
 type CreateEksPropertiesEks struct {
 	Version   string                     `json:"version,omitempty"`
 	NodePools map[string]NodePoolsAmazon `json:"nodePools,omitempty"`
+	Vpc       map[string]EksVpc          `json:"vpc,omitempty"`
```

Comment:
This shouldn't be a `map[string] EksVpc` but just a scalar `EksVpc` (`additionalProperties` shouldn't be used in the OpenAPI descriptor).